runtime/interpreter/value_test.go

```diff
@@ -2965,10 +2965,66 @@ func TestPublicKeyValue(t *testing.T) {
 		)
 
 		require.Equal(t,
-			"PublicKey(publicKey: [1, 7, 3], signatureAlgorithm: SignatureAlgorithm(rawValue: 2), isValid: true)",
+			"PublicKey(publicKey: [1, 7, 3], signatureAlgorithm: SignatureAlgorithm(rawValue: 2))",
 			key.String(),
 		)
 	})
+
+	t.Run("Panics when PublicKey is invalid", func(t *testing.T) {
```

Comment:
Test I added.

Reply:
Good idea to also add a unit test for the internal constructor function 👍